docs/events/jarsigning.md

```diff
@@ -39,30 +35,24 @@ that FML can work with the key.
 ### Checking at runtime
 To allow FML to compare the keys, add the public key to the `certificateFingerprint` argument in the `@Mod` annotation.
 
-```java
-    @Mod.EventHandler
-    public void onFingerprintViolation(FMLFingerprintViolationEvent event) {
-        
-    }
-```
-When FML detects that the keys don't match it will automatically execute the following event. How to handle this
-key mismatch is up the Modder.
+When FML detects that the keys don't match it will fire the mod-lifecycle event `FMLFingerprintViolationEvent` How to 
+handle this key mismatch is up to the developer.
 
-- `event.isDirectory()` - Returns true when the Mod runs in development environment.
+- `event.isDirectory()` - Returns true when the mod runs in development environment.
 - `event.getSource()` - Returns the file with the key mismatch.
 - `event.getExpectedFingerprint()` - Returns the public key.
 - `event.getFingerprints()` - Returns all public keys found.
 
 !!! note
 
-	The event is fired in mods that are already loaded and runs before any of the Mods code is executed.
+	The event runs before any of the Mod's code is executed.
```

Comment:
This doesn't really make any sense to me, and would probably be wrong for code in static initializers or that one really early FML phase event.